viz.yaml

```diff
@@ -86,20 +84,36 @@ fetch:
     refetch: TRUE
     metadata: "Updates every 6 hours on http://www.nhc.noaa.gov/gis/"
     comment: disabling refetch until timestamp is hooked up
+  -
+    id: precip-data
+    fetcher: precip
+    start.date: "2017-09-07"
+    end.date: "2017-09-08"
+    scripts: scripts/fetch/fetchPrecip.R
+    location: cache/precip.csv
+    mimetype: text/csv
+    refetch: true
+    depends:
+      counties: counties
```

Comment:
The PR looks good, but I don't follow this `thing1:thing1`, since the depends come back as a named list out of `readDepends()`